src/models/role.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const {
+  Model
```

Comment:
If it requires to destruct only one object, it's unnecessary to make it in 3 lines. Just make in one.
Same in other files

Reply:
yes